ang/civicase/case/details/configs/add-case-summary-blocks.config.js

```diff
@@ -0,0 +1,26 @@
+(function (angular) {
+  var module = angular.module('civicase');
+
+  module.config(function (CaseDetailsSummaryBlocksProvider) {
+    var showComingSoonCaseSummaryBlock = CRM['civicase-base'].showComingSoonCaseSummaryBlock;
+
+    if (!showComingSoonCaseSummaryBlock) {
+      return;
+    }
+
+    var caseSummaryBlocks = [
+      {
+        templateUrl: '~/civicase/case/details/summary-tab/case-details-summary-next-milestone.html',
+        weight: 0
+      }, {
+        templateUrl: '~/civicase/case/details/summary-tab/case-details-summary-next-non-milestone-activity.html',
+        weight: 0
+      }, {
+        templateUrl: '~/civicase/case/details/summary-tab/case-details-summary-calendar.html',
+        weight: 0
```

Comment:
why not use weight 0, 1, 2?

Reply:
Oops, that was a mistake. Used 0, 5, 10, so that items can be added in the middle.